tests/browser-automated-tests-playwright/tests/test_order_plan_with_only_upfront_charge.js

```diff
@@ -78,6 +78,13 @@ async function test_order_plan_with_only_upfront_charge(browsers, browserContext
     // Verify that plan is attached to subscriber
     const subscriber_plan_title_content = await page.textContent('.subscription-title');
     assert(subscriber_plan_title_content === 'One-Off Soaps');
+   
+   // Click Refresh Subscription
+    await page.click('#refresh_subscriptions'); // this is the refresh subscription
+    await page.textContent('.alert-heading') === "Notification";
```

Comment:
Same , but checking for different price https://github.com/Subscribie/subscribie/pull/573/files#r643543510